cmplibrary/src/main/java/com/sourcepoint/cmplibrary/util/FunctionalUtils.kt

```diff
@@ -25,20 +25,33 @@ internal fun <E> check(block: () -> E): Either<E> {
     }
 }
 
-internal fun <E> check(networkCode: NetworkCallErrorsCode? = null, block: () -> E): Either<E> {
+internal fun <E> check(
```

Comment:
Is it possible to rename the method? `check` is the most generic name I can think of. Maybe smth like `InovkeBlockAndConvertToConcreteException` or whatever is more specific than `check`?

Reply:
@Nevazhnovu this is a function that is being used throughout the whole application, so that will not be changed in this PR, but good call, we will get back at it when we work on TD